elkscmd/bootblocks/boot_minix.c

```diff
@@ -9,7 +9,7 @@
 
 // Global constants
 
-#define LOADSEG 0x1000
+#define LOADSEG 0x0100
```

Comment:
This is a risky change. I already mentionned in https://github.com/jbruchon/elks/issues/288#issuecomment-581034966 that some BIOSes are intolerant with LOADSEG not aligned to 64K. You should better keep loading the 'blob' to 0x1000:0 and then split it just after.

Reply:
Hello @mfld-fr, my suspicion is that at least some of the errors were happening because the reads were crossing a DMA boundary.  I had (earlier) modified `disk_read` to fall back to reading just 1 sector whenever a multi-sector read fails, which should allow such reads to work as long as the buffer is 512-byte aligned.

Reply:
Maybe I will see if I can tweak the code to read to `0:0x1000` (the same physical address) instead of `0x0100:0`.  This is relatively easy to do for the FAT bootloader, but a bit harder to do for the Minix FS bootloader.